examples/web_worker_prime/index.html

```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8">
+    <title>Yew • Web Worker Prime</title>
+
+    <link data-trunk rel="rust" href="Cargo.toml" data-bin="app" data-type="main" data-weak-refs />
+    <link data-trunk rel="rust" href="Cargo.toml" data-bin="worker" data-type="worker" data-weak-refs />
```

Comment:
This is because `.spawn()` and `.spawn_with_loader()`can provide proper tree shaking depending on which function is called. However, I do not think `loader={true}` will provide proper tree shaking.

I think a potential way to work it to provide it via a trait.
(a.k.a: `<WorkerProvider<Codec, ShimLoader> ...>` vs `<WorkerProvider<Codec, DefaultLoader> ...>`)

However, there is no good way of knowing whether tree shaking works unless actually try compiling it after implementing it. So I excluded it from this release.

I also feel it might be OK to require `data-shim-loader` for all agents given we properly document how the loader shim works.